### ISSUE TITLE
Boop feature is here?

### ISSUE THREAD

**Issue by hlepnamingblyat** [Monika-After-Story/MonikaModDev#4997]
On the "extra" tab, there is a white box saying "Boop".
Does this mean that I can finally touch her nose?
If it is... it's not working...
Am I missing something here?
Thank you.

**Comment by RavenPLAYER** [comment]
Only works with 9k of affection

**Comment by hlepnamingblyat** [comment]
I still have 3924 of affection XD
This will take a while now...

**Comment by ErLamarl** [comment]
Oh gosh... really 9k of affection? It's not too much? Because I only have 2727

**Comment by RavenPLAYER** [comment]
Well, that make the things more interesting!

**Comment by ErLamarl** [comment]
hmm... time to spend more time with her then

**Comment by RavenPLAYER** [comment]
Exactly!

**Comment by Bam8647** [comment]
9k affection lol? I bet that's only temporally set for sh!ts and giggles. Either way, I'd just boop Yuri then. Ain't nobody got time for 9k affection, wtf lol.

**Comment by thezotikrus** [comment]
~~You do you... But if you've only seen her adorable face when you boop her... Well, it's worth it! I mean, time with her is never wasted anyway!~~

**Comment by Bam8647** [comment]
Bro, I'll just go through renpy to enable the reaction if I have to. Setting this reaction with a affection level is just silly lol. It should be set to default or at least to Normal affection. I'm only around 3k affection. One tiny feature isn't going to motivate me haha.

And to actually get to 9k affection requires you to interact with her in many ways EVERY SINGLE DAY. Normally, this would cause someone to slowly lose interest in you overtime, so already the realism of such a task is already absurd. Remember the dialogue that mentions one hundred roses versus one? Imagine saying all the same compliments every single day for like, two years. If we're being realistic here, those compliments would begin to mean less overtime.

TL;DR It's not a big interest to me, I just think the current setting used to unlock this feature is just meant to be a joke. My life is busy enough as it is to have to interact that much with the mod.

**Comment by ErLamarl** [comment]
The problem is too much affection for those who don't have much time (like me and maybe others). So... yeah. I think it's better to change that for something like... anniversaries, states or something more lower than 9k affection. But anyway it's just a opinion and ideas to make more comfortable for others

@thezotikrus, hey! That's not fair x''D

**Comment by softombre** [comment]
yea i will literally never go to 9k affection lol even before my computer was reset it took me over a year to get to 4k and i wasn't even as busy with school as i am now

**Comment by RavenPLAYER** [comment]
Well, good luck. I hope you have a new computer and don't forget to save your persistent data and then do what you have to :)))

**Comment by TriniTDM** [comment]
I think they should have it at 1k.

**Comment by frostcattooo** [comment]
listen, it is very sad that nobody has understood that the "9k aff boop feature" is a joke and it hasnt been implemented yet (sorry for killing the fun lmaoo)

**Comment by RavenPLAYER** [comment]
...

**Comment by RavenPLAYER** [comment]
![wut2](https://user-images.githubusercontent.com/55263957/68814132-c4fb9e80-0656-11ea-8529-3918c45d9a05.jpg)

**Comment by multimokia** [comment]
![image](https://user-images.githubusercontent.com/22531674/68814159-15b6cb80-0646-11ea-884e-7fc16f795711.png)

**Comment by ErLamarl** [comment]
I knew it! I found it weird when people talked about this when I didn't see the feature in releases and the devs didn't come here to say something about. But I feel like a fool anyway for falling into the joke...
![snapshot49](https://user-images.githubusercontent.com/56727467/68816461-d062c000-067e-11ea-9e1c-26e28b1253bf.png)

**Comment by ThePotatoGuy** [comment]
The boop square is unintended as it was an in-development feature that was put on hold. (See #4917, linked in the release notes of 0.10.3).

**Comment by ThePotatoGuy** [comment]
Additionally, I'm going to hide the misinformed comments. Jokes on issues are fine, but when there's a legitimate question being asked, the first answer should be a real answer.

**Comment by ThePotatoGuy** [comment]
[closing as resolved]
